pkg/model/deployment_test.go

```diff
@@ -78,5 +90,6 @@ func TestOverrideBackstageImage(t *testing.T) {
 	assert.NoError(t, err)
```

Comment:
In L87 above, I'd suggest using [`t.Setenv`](https://pkg.go.dev/testing#T.Setenv) instead of `os.Setenv`. This will guarantee that the env var is restored to its original value after the test.

Reply:
ok